check-for-travis.rb

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env ruby
+require 'octokit'
+
+client = Octokit::Client.new :access_token => ENV['GITHUBTEACHER_TOKEN'] || raise("You need a GitHub Teacher access token")
+
+# Get the tree for a sha
+tree = client.tree("githubschool/on-demand-github-pages", "f1b10e926e7ace9862bf65b789204f22cccba1e0")
+
+# Select .travis.yml from the tree
+travis_yml = tree.tree.select { |t| t.path == ".travis.yml" }
```

Comment:
I think we need to verify other files besides `.travis.yml`, e.g. [the other test scripts in the repo](https://github.com/githubschool/on-demand-github-pages/tree/master/tests)?